src/main/java/seedu/eduke8/parser/MenuParser.java

```diff
@@ -13,43 +13,61 @@
 import seedu.eduke8.topic.TopicList;
 import seedu.eduke8.ui.Ui;
 
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 import static seedu.eduke8.exception.ExceptionMessages.ERROR_QUIZ_WRONG_FORMAT;
 import static seedu.eduke8.exception.ExceptionMessages.ERROR_UNRECOGNIZED_COMMAND;
 
+
 public class MenuParser implements Parser {
+    private static final Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
 
     public MenuParser() {
     }
 
     @Override
     public Command parseCommand(DisplayableList topicList, String userInput) throws Eduke8Exception {
+        assert topicList != null;
+
+        LOGGER.log(Level.INFO, "Begin parsing command.");
         String[] commandArr = userInput.trim().split(" ", 0);
 
         Ui ui = new Ui();
         switch (commandArr[0]) {
         case "about":
+            LOGGER.log(Level.INFO, "Parsing complete: about command chosen.");
             return new AboutCommand();
         case "help":
+            LOGGER.log(Level.INFO, "Parsing complete: help command chosen.");
             return new HelpCommand();
         case "topics":
+            LOGGER.log(Level.INFO, "Parsing complete: topics command chosen.");
             return new TopicsCommand();
         case "textbook":
+            LOGGER.log(Level.INFO, "Parsing complete: textbook command chosen.");
```

Comment:
Good idea to log this but maybe using constants can help since most of the words in these strings are the same. Will open an issue to use more OOP for v2

Reply:
Roger, this makes a lot of sense. Thank you!

Reply:
Yup, I agree! I think most of us will have to do that too and that could fall under an issue for v2.0.